.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "liveServer.settings.port": 5501
+    "liveServer.settings.port": 5502
 }
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly add this folder to the gitignore file as this folder is not relevant for this project.

Reply:
- This issue is not yet resolved :x: